main.go

```diff
@@ -242,16 +240,6 @@ func checkConfig(v *viper.Viper) error {
 		if len(environment) == 0 {
 			return errors.New("when status is set to STOPPED then environment must be set")
 		}
-		valid := false
```

Comment:
I totally get why you don't want to be checking against a bunch of strings that were simply names we decided for, but is there any other way to check that we are using the environments we anticipated? What happens when someone passes in an unusable env? Do we catch it elsewhere?

Reply:
If you put in a name that doesn't exist you'll get an error say as such
```
panic: error describing log streams: ResourceNotFoundException: The specified log group does not exist.
```